ui/app/templates/vault/cluster/access/oidc/assignments/assignment/details.hbs

```diff
@@ -61,25 +61,23 @@
 <div class="box is-fullwidth is-sideless is-paddingless is-marginless">
   <InfoTableRow @label="Name" @value={{@model.name}} />
   <InfoTableRow
+    @label="Entities"
     @type="array"
     @value={{@model.entityIds}}
     @model={{@model}}
     @isLink={{true}}
-    @label="Entities"
     @modelType="oidc/assignment"
-    @viewAll="entities."
```

Comment:
decided to remove since this arg seemed redundant when we already pass in `@label`

Reply:
Using the label makes sense to me.